client/src/components/common/names-capture.vue

```diff
@@ -3,7 +3,7 @@
     <v-container fluid class="pa-0" id="send-to-examination-container">
       <template v-if="editMode">
         <v-row justify="end" class="mt-n3 mb-n5">
-          <v-col cols="6" class="copy-normal">I need a name to:</v-col>
+          <v-col cols="6" class="copy-normal copy-bold">I need a name to:</v-col>
```

Comment:
Here's hoping this project moves to the existing Vuetify helper classes in the near future.
https://vuetifyjs.com/en/styles/text-and-typography/#font-emphasis etc

Reply:
Absolutely, i tried to play it safe in this ticket. The scope was already large :)

Reply:
How about this? If you're in there again, tweak it, otherwise it will a future, global effort.

Reply:
Absolutely, good plan!